### PR TITLE
update userguide to include introduction and overview

### PR DESCRIPTION
**Pull request by lawwman**

### CODE REVIEW

**Review by khooroko — APPROVED** [review]

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -14,6 +14,22 @@ endif::[]
 
 By: `CS2103Aug2017 T17-B1`      Since: `Sep 2017`      Licence: `MIT`
 
+== About
+
+This user guide will give you an overview of how to properly use the Codii app.
```
Comment (jelneo) [inline docs/UserGuide.adoc]: i think the word 'properly' is unnecessary

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -14,6 +14,22 @@ endif::[]
...
+This guide includes step by step instructions on how each commands work, FAQs and a summary on every command you will learn.
```
Comment (jelneo) [inline docs/UserGuide.adoc]: - step-by-step*
- each command*
- summary of*

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -14,6 +14,22 @@ endif::[]
...
+
+The terms used in this guide are as followed:
```
Comment (jelneo) [inline docs/UserGuide.adoc]: follows*

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -14,6 +14,22 @@ endif::[]
...
+* Blacklist - List of people who are banned
+* Index - The ordering of a contact on the list
```
Comment (jelneo) [inline docs/UserGuide.adoc]: in* the list

**Review by jelneo — APPROVED** [review]

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -14,6 +14,23 @@ endif::[]
 
 By: `CS2103Aug2017 T17-B1`      Since: `Sep 2017`      Licence: `MIT`
 
+== About
+
+This user guide will give you an overview of how to use the Codii app.
```
Comment (jaivigneshvenugopal) [inline docs/UserGuide.adoc]: how to use Codii*

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -14,6 +14,23 @@ endif::[]
...
+This guide includes step-by-step instructions on how each command work, FAQs and a summary of every command you will learn.
+
+The terms used in this guide are as follows: +
+
+* Blacklist - List of people who are banned +
+* Index - The ordering of a contact in the list
+
+== Introduction
+
+Codii is a free to use address book catered to suit the needs of debt collectors. You can enter debtors into the Codii app,
```
Comment (jaivigneshvenugopal) [inline docs/UserGuide.adoc]: debtors into Codii*

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -85,7 +102,7 @@ Examples:
 
 === Listing all persons : `list`
 
-Shows a list of all persons in the address book. +
+Shows a list of all contacts in the address book. +
```
Comment (jaivigneshvenugopal) [inline docs/UserGuide.adoc]: debtors? instead of contacts.

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -95,7 +112,7 @@ Format: `blacklist`
 
 === Editing a person : `edit`
 
-Edits an existing person in the address book. +
+Edits an existing debtor, black / white listed contact in the address book. +
```
Comment (jaivigneshvenugopal) [inline docs/UserGuide.adoc]: blacklisted/whitelisted*

**Review by jaivigneshvenugopal — APPROVED** [review]

**Pull request merged**